### ISSUE TITLE
Diagnostic info text is displayed in the code action picker

### ISSUE THREAD

**Issue by CptPotato** [helix-editor/helix#794]
LSP information text that is currently displayed seems to be added to the code action picker? 😕

![image](https://user-images.githubusercontent.com/3957610/135411338-79f97bfb-8ed6-4c73-95d5-e4fcd764f22c.png)

### Reproduction steps

(see screenshot)

### Environment

- Platform: Win10 x64
- Helix version: v0.4.1-191-gd68cff8

**Comment by Omnikar** [comment]
Well, changing `x` to `_x` to fix a warning *is* a code action, isn't it?

**Comment by CptPotato** [comment]
Maybe that's what's happening here?

I encountered it in a more complex scenario got confused when I saw wordy lines of text that were cut off and I couldn't make any sense of. Something like this which seemd out of place for the code action picker:

![image](https://user-images.githubusercontent.com/3957610/135745034-797d7f70-3494-4f40-8655-795e89f9e197.png)

I also can't execute any of those specific code actions but it appers there's already an issue for that.
